### PR TITLE
Add p50, p90, and p99 Percentile Calculations to Grafana HTTP Requests Dashboard

### PR DESCRIPTION
**Pull request by Iajrdev**
#### Overview:
This pull request introduces percentile calculations (`p50`, `p90`, `p99`) to the HTTP Requests dashboard in Grafana, enhancing the ability to monitor and analyze request distribution.

#### Changes Made:
- **Dashboard Configuration:** Updated `dashboard.json` to include queries for `p50`, `p90`, and `p99` percentiles using the `histogram_quantile` function from Prometheus.
- **Visualization:** Adjusted the dashboard panels to ensure the percentiles are displayed prominently and accurately, offering a detailed view of request durations.

image

**P50**
image

**P90**
image

**p99**
image

### CODE REVIEW

**Review by alipourhabibi — APPROVED** [review]
LGTM

**Review by alipourhabibi — APPROVED** [review]
LGTM

File: grafana/provisioning/dashboards/dashboard.yml
Comment (armineyvazi) [inline grafana/provisioning/dashboards/dashboard.yml:1]: ### Issue in `dashboard.yml`

The `dashboard.yml` file currently specifies an incorrect path in the `options` section under `providers`.

**Current Configuration:**
```yaml
options:
  path: /grafana/provisioning/dashboards
```

**Problem:**
The path `/grafana/provisioning/dashboards` does not exist. The correct path that should be used is `/etc/grafana/provisioning/dashboards`.

**Suggested Fix:**
Update the `path` in `dashboard.yml` to the correct directory:
```yaml
options:
  path: /etc/grafana/provisioning/dashboards
```
@Iajrdev

Comment (Iajrdev) [reply grafana/provisioning/dashboards/dashboard.yml:1]: Fixed

**Review by armineyvazi — APPROVED** [review]

**Pull request merged**